libs/examples/tslint.json

```
@@ -0,0 +1,7 @@
+{
+  "extends": "../../tslint.json",
+  "rules": {
+    "directive-selector": [true, "attribute", "dtExample", "camelCase"],
```

you are using the camelCase attribute ;)
```suggestion
    "directive-selector": [true, "attribute", "dt-example", "camelCase"],
```

---

camelCase is correct. Why would you wanna change it?

---

you have to write the lint rule as followed ;)

---

sorry you are right :)